File: travel-planner-frontend/src/models/Expense.js
export class Expense {
  constructor(id, planId, name, category, amount, date, description) {
    this.id = id;
    this.planId = planId;
    this.name = name;
    this.category = category;
    this.amount = amount;
    this.date = date;
    this.description = description;
  }
}

File: TravelPlanner.ServiceFabric/TravelPlanner.ExpenseService/Models/Expense.cs
namespace TravelPlanner.ExpenseService.Models;

public class Expense
{
    public Guid Id { get; set; }
    public Guid PlanId { get; set; }
    public string Name { get; set; } = string.Empty;
    public string Category { get; set; } = string.Empty; // transport, accommodation, food, tickets, shopping, other
    public decimal Amount { get; set; }
    public DateTime Date { get; set; }
    public string Description { get; set; } = string.Empty;
    public DateTime CreatedAt { get; set; }
}